source/tutorials/deploying-nixos-using-terraform.rst

```diff
@@ -0,0 +1,179 @@
+.. _deploying-nixos-using-terraform:
+
+Deploying NixOS using Terraform
```

Comment:
`s/Terraform/Terraform Cloud/`

While it can probably be translated to the non cloud version this would be more correct right now.

Reply:
That's not really correct, since it doesn't run on Cloud but it only stores the state there.